CMakeLists.txt

```diff
@@ -23,7 +39,9 @@ include_directories(third-party/miniupnp)
 
 find_package(Threads REQUIRED)
 find_package(OpenSSL REQUIRED)
-set(Boost_USE_STATIC_LIBS ON)
```

Comment:
I'm wondering if this is the reason for the build failures. There's a reference to `boost` directly before the error message.

https://github.com/SunshineStream/Sunshine/runs/5325277331?check_suite_focus=true#step:4:2515
https://github.com/SunshineStream/Sunshine/runs/5325278122?check_suite_focus=true#step:4:3114